t.sh

```diff
@@ -2,10 +2,14 @@
 #
 # Outer wrapper for invoking test.sh inside docker-compose.
 #
+
 if type realpath >/dev/null 2>&1 ; then
   cd "$(realpath -- $(dirname -- "$0"))"
 fi
+
+source ./tools/t-helper.sh
+
 # Use a predictable name for the container so we can grab the logs later
 # for use when testing logs analysis tools.
 docker rm boulder_tests
-exec docker-compose run --name boulder_tests boulder ./test.sh "$@"
+${_compose} run --name boulder_tests boulder ./test.sh "$@"
```

Comment:
Ah, this is a good catch. For some history, we have this `exec` here because otherwise we encounter a 10s hang when using ctrl+c to stop tests which have yet to conclude.

Reply:
Ah, I will put that back in. That's not something that I had encountered so I thought it was unnecessary.